src/main/java/org/folio/ld/dictionary/model/Resource.java

```diff
@@ -17,6 +17,7 @@
 @NoArgsConstructor
 @Accessors(chain = true)
 @EqualsAndHashCode(of = "id")
+@SuppressWarnings("javaarchitecture:S7027")
```

Comment:
try use //NOSONAR instead

Reply:
Per this doc, we should not use NOSONAR -> https://dev.folio.org/guides/code-analysis/